.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,58 @@
+name: "\U0001F41BBug report"
+description: Create a report to help us improve Optuna.
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! Please write a clear and concise description of what the bug is.
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected behavior
+      description: Please write a clear and concise description of what you expected to happen.
+    validations:
+      required: true
+  - type: textarea
+    id: environment
+    attributes:
+      label: Environment
+      description: |
+        Please give us some environment information. You can get this information by typing the following.
+        ```
+        python -c 'import optuna; print(optuna.__version__)'
+        python -c 'import platform; print(platform.python_version())'
+        python -c 'import platform; print(platform.platform())'
+        ```
+    validations:
```

Comment:
The default value seems to be missing.
```suggestion
        ```
      value: |
        - Optuna version:
        - Python version:
        - OS:
        - (Optional) Other libraries and their versions:
    validations:
```